### PR TITLE
cargo-expand support specified function

### PR DESCRIPTION
**Pull request by light4**
`:expand`: `cargo-expand` support specified function

### CODE REVIEW

**Comment by sigmaSd** [comment]
Thanks !

**Comment by sigmaSd** [comment]
The packaging failure is unrelated to your pr

**Comment by sigmaSd** [comment]
Can you explain how to use this ? this is what I tried
image


Also I tried adding macro_export but same result

**Comment by light4** [comment]
It works like `:asm function`. Must be followed by a `pub fn`.
图片

**Comment by sigmaSd** [comment]
I added a commit that should improve ux https://github.com/sigmaSd/IRust/commit/5739c92366eaa6109b22b8d7ffde2636afcaa840

Previously without this, if the user had a statement like `let a =4;` it would corrupt the repl.

Please rebase (just to trigger ci)

File: crates/irust/README.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ You can try out IRust with no installation or setup (via Gitpod.io) by visiting
 
 **:dbg** *\<expression\>* => Spawn rust-lldb/rust-gdb with (an optional expression), example: `:dbg` or `:dbg fact(12)`, The debugger can be specified in the config file
 
-**:expand** => Shows the result of macro expansion, requires https://github.com/dtolnay/cargo-expand
+**:expand** *\[function\]* => Shows the result of macro expansion, requires https://github.com/dtolnay/cargo-expand, function is optional
```
Comment (sigmaSd) [inline crates/irust/README.md:59]: I think we should also say "example `fn b() {println!("4");}` `:expand b`"

Comment (light4) [reply crates/irust/README.md]: updated

**Pull request merged**